variables.tf

```diff
@@ -87,34 +87,27 @@ variable "key_protect_endpoint_type" {
   }
 }
 
-variable "existing_key_protect_instance_guid" {
+variable "existing_kms_instance_guid" {
   type        = string
-  description = "The GUID of an existing Key Protect instance, required if 'var.create_key_protect_instance' is false."
+  description = "The GUID of an existing Key Protect or Hyper Protect Crypto Services instance, required if 'var.create_key_protect_instance' is false."
   default     = null
 }
 
 variable "key_map" {
```

Comment:
its no longer a map, so suggest renaming the variable. Maybe just call it `keys` ?